lib/elixir_bench/github/client_in_memory.ex

```diff
@@ -0,0 +1,43 @@
+defmodule ElixirBench.Github.ClientInMemory do
+  @moduledoc """
+  The Github InMemory client.
+
+  This client is a mock module to be used in tests.
+
+  This module implements the following functions:
+
+    - a `get_yaml/1` function that takes the url path of the project and return
+      a map with mocked data about the raw configuration found in the
+      benchee.yml file.
+  """
+
+  @behaviour ElixirBench.Github.Client
+
+  def get_yaml(_) do
+    {:ok,
+      %{
```

Comment:
I believe the indentation is slightly wrong here, if you wanted you could run `mix format` (in Elixir >= 1.6) on just this file to fix it :)